misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlLexer.java

```diff
@@ -383,14 +399,14 @@ private STNode processEndOfLine() {
         char c = reader.peek();
         switch (c) {
             case LexerTerminals.NEWLINE:
-                reader.advance();
-                return STNodeFactory.createMinutiae(SyntaxKind.END_OF_LINE_MINUTIAE, getLexeme());
+//                reader.advance();
+                return STNodeFactory.createMinutiae(SyntaxKind.END_OF_LINE_MINUTIAE, "\n");
             case LexerTerminals.CARRIAGE_RETURN:
-                reader.advance();
-                if (reader.peek() == LexerTerminals.NEWLINE) {
-                    reader.advance();
-                }
-                return STNodeFactory.createMinutiae(SyntaxKind.END_OF_LINE_MINUTIAE, getLexeme());
+//                reader.advance();
+//                if (reader.peek() == LexerTerminals.NEWLINE) {
+//                    reader.advance();
+//                }
```

Comment:
Need to handle the carriage return properly.
We should emit only one newline token regardless whether it is `\n` or `\r\n`

Reply:
Should be okay now.